code/modules/antagonists/fugitive/fugitive.dm

```diff
@@ -0,0 +1,14 @@
+/datum/antagonist/fugitive
+	name = "Fugitive"
+	show_in_antagpanel = FALSE
+	roundend_category = "Fugitive"
+	silent = TRUE //greet called by the event
+
+/datum/antagonist/fugitive/greet(backstory)
+	to_chat(owner, "<span class='boldannounce'>You are the Fugitive!</span>")
+	switch(backstory)
+		if("prisoner")
+			to_chat(owner, "<B>I can't believe we managed to break out of a Nanotrasen superjail! Sadly though, our work is not done. The emergency teleport at the station logs everyone who uses it, and where they went.</B>")
+			to_chat(owner, "<B>It won't be long until Centcom tracks where we've gone off to. I need to work with my fellow escapees to prepare for the troops Nanotrasen is sending, I'm not going back.</B>")
+	to_chat(owner, "<span class='boldannounce'>You are not an antagonist in that you may kill whoever you please, but you may do anything to avoid capture.</span>")
```

Comment:
Should it be "whoever" or "whomever"? I always forget, but you might wanna check to see if you used the right one.

Reply:
I am really terrible at writing fluff, I need to go over this whole thing anyways. If someone wants to write some actually good backgrounds, i'd be grateful.

Reply:
It's supposed to be whomever. Trick to remember the difference: Replace "who" with "he" or "she" and "him" or "her". If he/she sounds right, it's who. If him/her sounds right, it's whom. "...you may kill he..." is wrong, but "...you may kill him..." is correct.

Nobody gives a shit if you mix the two up though.

Reply:
thanks for the help, i fixed it!